### ISSUE TITLE
Still not executing the correct script after changing Stage Counter Similarity

### ISSUE THREAD

**Issue by advancedorange** [Fate-Grand-Automata/FGA#772]
Before opening an issue:
- [x] I tested the latest release
- [x] I looked at other issues (even the closed ones)
- [x] I read the [Troubleshooting Guide](https://github.com/MathewSachin/Fate-Grand-Automata/wiki/Troubleshooting)
- [x] I'll fill this template properly

> Your issue will get closed/locked by the bot in case proper info isn't provided
If you open unnecessary/dumb issues, you'll be rewarded with appropriate tags.

**Describe the bug**
The old report got closed, but the issue started reappearing

Previous description:
A few days ago I switched to a new node to farm and created a new script, but this script does not what its supposed to do.
Most of the time the script for wave 3 gets activated on wave 2, sometimes the script even executes the steps for wave 2 on the second enemy on the first wave, followed by the steps for wave 3 on the third one (still wave 1). One time this was happening I think I noticed FGA trying to tap the skill for wave 2 right after it pressed the attack button for wave 1 (before the cards even popped up).

Setting the Stage Counter Similarity to 75% did not fully fix this problem
Today I once again had a run during which everything was executed on the first wave

Script: > i h g1 d ,#, f 5 ,#, g1 i h k1 c 4 ,#, 
Party: Summer Scathach, Yu Meiren, Waver 
Node: Fallen Babylon
Server: NA
Link to reddit: [Here](https://old.reddit.com/r/grandorder/comments/nyvt9g/help_and_question_thread_june_13_2021/h29ir3m/ )

**Screenshots**
x

**Video**
x

**Device Info (please fill at-least 'Device model'):**
 - Device model: [Honor 8]
 - Screen size: [1920x1080]
 - Android Version: [Android 8 (?)]
 - RAM: [4GB]

**Comment by reconman** [comment]
> One time this was happening I think I noticed FGA trying to tap the skill for wave 2 right after it pressed the attack button for wave 1 (before the cards even popped up).

Sounds like you need to increase `Wait after click`.

**Comment by MathewSachin** [comment]
> **Screenshots**
If applicable, add screenshots to help explain your problem.
It is recommended that you at-least add one screenshot of FGO running on your device.
>
>**Video**
If relevant, include a video of the script running on your device with `Debug mode` ON
Recording functionality is available within the app in `More options`.

☝️

**Comment by advancedorange** [comment]
How much should I increase Wait after click?

I'll try to make a video tomorrow

**Comment by reconman** [comment]
> How much should I increase Wait after click?

You'll have to experiment.

**Comment by advancedorange** [comment]
Since this was only in 1 out of many runs I don't really have a way to experiment with this specific situation.

Managed to get a video of the script messing up stages though.
This video has 2 runs: The first one is as it should be, the second one has the errors.
-Yu Meirens 3rd skill (and NP) should have been activated on wave 2, got executed on wave 1
-Boosts for Scathach are meant for wave 3, not wave 2
After those errors happened and the script was "finished" I manually selected NPs to speed up the run
I tried to get a completely messed up run (everything executed on wave 1) on video, but those are really rare and I did not get one the last days


[Link](https://old.reddit.com/user/Advanced_Orange2293/comments/oha9gv/fga_video/)
At the time of posting this the video is still processing, should be available soon though
The 2nd run starts around 3:40
